addons/l10n_ar/tests/common.py

```diff
@@ -9,7 +9,6 @@
 _logger = logging.getLogger(__name__)
 
 
-@tagged('external_l10n', '-at_install', 'post_install', '-standard', 'external')
```

Comment:
@jco-odoo 
Hi Jos
I did not understood why remove this decoratotr? is because is a common class?

Reply:
yes and it might be handled differently in 16 as in 15